packages/vendure-plugin-primary-collection/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@pinelab/vendure-plugin-primary-collection",
+  "version": "0.0.1",
```

Comment:
You can make this 1.0.0 if you'd like. It looks finished enough :mechanical_arm: